src/structures/ThreadChannel.js

```diff
@@ -275,11 +275,20 @@ class ThreadChannel extends Channel {
    *   .catch(console.error);
    */
   async edit(data, reason) {
+    let maxAutoArchiveDuration;
```

Comment:
```suggestion
    let maxAutoArchiveDuration = data.autoArchiveDuration;
```
and remove the `?? ...` at the end